src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
-Title: Managed solutions under SLA
+Title: Managed solutions
 
-# Managed solutions under SLA
+# Managed solutions
 
-[Juju experts](https://jujucharms.com/experts) may designate a Juju
-model as a managed, supported solution by setting an SLA.
+A Juju Expert is a partner backed by professional support at Canonical Ltd.
+Such a partner deploys and supports an infrastructure based on Juju. We call
+this a *managed solution*.
 
-There are three support levels that may be set on a model:
+Any support requests beyond the build, workload, and operation of a particular
+solution can be escalated to Canonical by means of their Ubuntu Advantage
```

Comment:
so this is written to the end user who might work with an expert. Does the expert do the escalation or the end user?

Reply:
The Expert. I will specify.

Reply:
So if this is the expert why put it here in end user documentation? They're not going to do the escalating and it's in the Experts area of wrapping things for the end user. This seems like sharing implementation details. We can just say we support the experts and you can trust them.

Reply:
Knowing that the Expert can escalate to Canonical, but only for certain things, is a huge part of the deal. More transparency.